### PR TITLE
IDEV-1899: Implement nod list

### PR DESCRIPTION
**Pull request by jbabac**
action demo: https://www.loom.com/share/394de0d275f84e61a56bd1fb4d8d2884

playbook demo: https://www.loom.com/share/3f4b578d5afc423680edb8d04d6b9a55

### CODE REVIEW

File: .DS_Store
Comment (briluza) [inline .DS_Store:1]: should be removed, no?

Comment (jbabac) [reply .DS_Store:1]: oops. this has been removed already.

**Review by wesleya — APPROVED** [review]
good to go! Do we need to fix the linting errors?

**Comment by jbabac** [comment]
> good to go! Do we need to fix the linting errors?

@wesleya Thank you. yes, I'm still working on it

**Review by briluza — APPROVED** [review]
LGTM ✅

**Pull request merged**